gradle-plugin/src/main/groovy/io/realm/gradle/Realm.groovy

```diff
@@ -76,4 +83,19 @@ class Realm implements Plugin<Project> {
             return false
         }
     }
+
+    private static boolean shouldApplyAndroidAptPlugin(boolean usesAptPlugin, boolean usesKotlinPlugin,
+                                                       boolean hasAnnotationProcessorConfiguration,
+                                                       boolean preferAptOnKotlinProject) {
+        if (usesAptPlugin) {
+            // for any projects that use `android-apt` plugin
```

Comment:
I assume you mean `for any projects that uses android-apt plugin already. No need to apply it twice`. Just to clarify it? Because just reading it at a glance looks a bit weird.